i18n/nl/cosmic_store.ftl

```diff
@@ -5,11 +5,11 @@ install = Installeren
 check-for-updates = Naar bijwerkingen zoeken
 checking-for-updates = Op zoek naar bijwerkingen...
```

Comment:
"Op zoek naar bijwerkingen..." is a weird sentence, because "bijwerkingen" are  unmeant symptoms of prescribed drugs.
line 5 could be: "Naar updates zoeken" and
line 6 could be: "Zoeken naar updates..." or "Op zoek naar updates..."